prettier.config.js

```diff
@@ -1,11 +1,9 @@
 const config = {
-	organizeImportsSkipDestructiveCodeActions: true,
 	printWidth: 120,
 	semi: true,
 	singleQuote: true,
 	trailingComma: 'all',
 	useTabs: true,
-	plugins: ['prettier-plugin-organize-imports'],
```

Comment:
Removing prettier plugin to let `eslint` organize imports as groups are defined on eslint config file